libvirt.go

```diff
@@ -39,6 +39,22 @@ import (
 // are unsupported by either QEMU or libvirt.
 var ErrEventsNotSupported = errors.New("event monitor is not supported")
 
+// LibvirtURI defines a type for driver URIs for libvirt
+// the defined constants are *not* exhaustive as there are also options
+// e.g. to connect remote via SSH
+type LibvirtURI string
```

Comment:
I'm OK with using a type here, but the name stutters: it's libvirt.LibvirtURI. I suggest shortening the type name to just URI.

Reply:
what about `DriverURI` to avoid confusion?

Reply:
`ConnectURI`? naming is the worst.

Reply:
:joy: true that - I'll change it to `ConnectURI`